tests/test_hdfs.py

```diff
@@ -9,16 +10,22 @@
 import sdk_cmd
 import sdk_hosts
 import sdk_install
+import sdk_marathon
+import sdk_tasks
 
 from tests import utils
 
 
 log = logging.getLogger(__name__)
 DEFAULT_HDFS_TASK_COUNT=10
 GENERIC_HDFS_USER_PRINCIPAL = "hdfs@{realm}".format(realm=sdk_auth.REALM)
+KEYTAB_SECRET_PATH = "__dcos_base64___keytab"
```

Comment:
may want to make this optionally assignable with an environment variable when we eventually want to run this in soak...

Reply:
Yes, good point.